### ISSUE TITLE
Create side filterbutton

### ISSUE THREAD

**Issue by lindacnatt** [osqledaren/osqledaren-frontend#34]
Create the filterbutton that will indicate what category the article belongs to and that will direct you to the a page with other articles in the same category

**Comment by johannesloor** [comment]
The title of the article is now the same color as the category it belongs to, so maybe this could be skipped?

**Comment by lindacnatt** [comment]
Yes, maybe it's just redundant. I think the idea was to have some sort of connection to the layout of the paper. But yes, it's not so necessary to have.
